internal/core/metadata/rest_addressable.go

```diff
@@ -35,9 +36,10 @@ func restGetAllAddressables(
 	w http.ResponseWriter,
 	loggingClient logger.LoggingClient,
 	dbClient interfaces.DBClient,
-	errorHandler errorconcept.ErrorHandler) {
+	errorHandler errorconcept.ErrorHandler,
+	config config.ConfigurationStruct) {
 
-	op := addressable.NewAddressableLoadAll(Configuration.Service, dbClient, loggingClient)
+	op := addressable.NewAddressableLoadAll(config.Service, dbClient, loggingClient)
```

Comment:
To be consistent with other refactorings, use `configuration` instead of `config` for those variables that you're passing down the call chain to replace the now-removed `Configuration` global.

Mentioned here; occurs elsewhere.